Documentation/DeterministicBuild.md

```diff
@@ -0,0 +1,50 @@
+# Deterministic build
+
+Deterministic build **is supported only** by `msbuild`(`/p:CollectCoverage=true`) and `collectors`(`--collect:"XPlat Code Coverage"`) drivers.
+
+Deterministic builds are important as they enable verification that the resulting binary was built from the specified source and provides traceability.  
+For more information on how to enable deterministic build I recommend you to take a look at [@clairernovotny](https://github.com/clairernovotny)'s guide https://github.com/clairernovotny/DeterministicBuilds
+
+From coverage perspective deterministic build put some challenge because usually coverage tools need access to source file metadata(ie. local path) during instrumentation and report generation.  
```

Comment:
```suggestion
From coverage perspective deterministic build put some challenge because usually coverage tools need access to source file metadata (ie. local path) during instrumentation and report generation.  
```